packages/ui-extensions/src/surfaces/customer-account/api/order-status/order-status.ts

```diff
@@ -1194,4 +1426,9 @@ export interface Order {
    * If cancelled, the time at which the order was cancelled.
    */
   cancelledAt?: string;
+
+  /**
+   * A randomly generated alpha-numeric identifier for the order.
+   */
+  confirmationNumber?: string;
```

Comment:
it's potentially undefined on the host tho. https://github.com/Shopify/customer-account-web/pull/3459/files#diff-4498d070259fc7036860e7044465ed6b2656db2516c77d1b1aa4679915ba438cR17

Reply:
Interesting yeah 🤔 
> that may be shown to the customer instead of the sequential order name

Reply:
> Is there any reason to not copy paste the exact GraphQL description for the field?

not sure if that's a good excuse, but I just copied it from the other PR to make it's consistent with the backported version. 😅

Reply:
Just to confirm from @thitemple that we are keeping this?

Reply:
There's not yet a decision made on this, the discussion can be followed [here](https://github.com/Shopify/checkout-web/discussions/28848). I'd say it will be better to move without it for now and add it later

Reply:
any updates @thitemple ?

Reply:
There’s no decision made yet and I wouldn't keep it in this PR, I believe there's a chance we may use a different name other than `orderIdentity`.
The discussion is still ongoing [here](https://github.com/Shopify/checkout-web/discussions/28848)